### PR TITLE
Refactor: remove cui-labels

### PR DESCRIPTION
**Pull request by ganeshkumar1989**
SCDC-919

### Requirements

* The cui-labels component has to be removed and replaced with oui-chip component.

### Description of the Change

* The cui-labels component has been replaced with oui-chip
* The cui-labels component has been removed completely.

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report build-yarn#5536.0 ✘  
* Stage 1
  * deps ✔ 
* Stage 2
  * build ✘  
  * test ✘

**Comment by ovh-cds** [comment]
CDS Report build-yarn#5537.0 ✘  
* Stage 1
  * deps ✔ 
* Stage 2
  * build ✘  
  * test ✘

File: client/app/iplb/vrack/iplb-vrack.html [hunk 1]
```diff
@@ -50,7 +50,7 @@
         <oui-column data-title="'iplb_vrack_list_col_subnet' | translate" data-property="subnet" data-sortable></oui-column>
         <oui-column data-title="'iplb_vrack_list_col_nat_ip' | translate" data-property="natIp" data-sortable></oui-column>
         <oui-column data-title="'iplb_vrack_list_col_farms' | translate">
-            <cui-labels data-items="$row.farmId">{{ $item.displayName || $item.farmId }}</cui-labels>
+            <span class="oui-chip" data-ng-repeat="farm in $row.farmId">{{ farm.displayName || farm.farmId }}</span>
```
Comment (antleblanc) [inline client/app/iplb/vrack/iplb-vrack.html]: Prefer using `ngBind`:)

File: client/app/iplb/vrack/iplb-vrack.html [hunk 2]
```diff
@@ -50,7 +50,7 @@
...
         </oui-column>
         <oui-action-menu align="end" compact>
```
Comment (antleblanc) [inline client/app/iplb/vrack/iplb-vrack.html]: Could you please add missing `data-` prefix?

File: client/app/iplb/vrack/iplb-vrack.html (hunk 1 above)
Comment (ganeshkumar1989) [reply client/app/iplb/vrack/iplb-vrack.html]: This is done :)

File: client/app/iplb/vrack/iplb-vrack.html (hunk 2 above)
Comment (ganeshkumar1989) [reply client/app/iplb/vrack/iplb-vrack.html]: data- prefix is already there for the angular attributes... class is a native attribute and hence does not have the prefix. Am I missing something else?

Comment (antleblanc) [reply client/app/iplb/vrack/iplb-vrack.html]: @ganeshkumar1989 you're right.

Here I'm targeting the `align` and `compact` attributes which are attributes related to the `<oui-action-menu>` component (according to the API table from the documentation http://master.ui-kit.ovh/#!/oui-angular/action-menu).

File: client/app/iplb/vrack/iplb-vrack.html [hunk 3]
```diff
@@ -50,7 +50,7 @@
         <oui-column data-title="'iplb_vrack_list_col_subnet' | translate" data-property="subnet" data-sortable></oui-column>
         <oui-column data-title="'iplb_vrack_list_col_nat_ip' | translate" data-property="natIp" data-sortable></oui-column>
         <oui-column data-title="'iplb_vrack_list_col_farms' | translate">
-            <cui-labels data-items="$row.farmId">{{ $item.displayName || $item.farmId }}</cui-labels>
+            <span class="oui-chip" data-ng-repeat="farm in $row.farmId" data-ng-bind="farm.displayName || farm.farmId"></span>
```
Comment (antleblanc) [inline client/app/iplb/vrack/iplb-vrack.html]: +`track by`?

Comment (ganeshkumar1989) [reply client/app/iplb/vrack/iplb-vrack.html]: This is done :)

**Pull request merged**